cli/tests/program.rs

```diff
@@ -824,7 +824,7 @@ fn test_cli_program_upgrade_auto_extend() {
         skip_fee_check: false,
         compute_unit_price: None,
         max_sign_attempts: 5,
-        no_extend: true, // --no-extend (true)
+        auto_extend: false, // --no-extend present (true)
```

Comment:
tiny nit on the comment:

```suggestion
        auto_extend: false, // --no-auto-extend flag is present (true)
```